### DISCUSSION TITLE
.mp4 background support for next.js CSS-in-JS

### DISCUSSION THREAD

**Discussion by stevenrugg** [vercel/next.js#43523] [Help]
Is there support for attaching videos to the background using css?
![image](https://user-images.githubusercontent.com/43973132/204572733-8ada340b-21b7-4a43-85ac-d669244de818.png)

This code compiles correctly and the video is in the public folder so I shouldn't have to import it right? Its not rendering the .mp4 video background in the latest versions of FF, Chrome or Edge.

**Comment by icyJoseph** [comment]
AFAIK, background doesn't work with videos, only images and colors, roughly speaking. Use a video tag instead.

Note that this is a CSS specification. Nothing to do with CSS-in-JS.
